tests/test_api_tests_pytest.py

```diff
@@ -0,0 +1,118 @@
+import pytest
+from hamcrest import *
+import requests
+import datetime
+import time
+
+
+def test_get_list_users():
+    # setup
+    get_list_users_url = "https://reqres.in/api/users?page=2"
+    # actions
+    response = requests.get(url=get_list_users_url)
+    # results
+    assert_that(response.status_code, is_(equal_to(200)), reason="Wrong status code")
+    assert_that(response.json()["data"][0]["email"], is_(equal_to("michael.lawson@reqres.in")), reason="Wrong email")
+    assert_that(response.json()["data"][0]["first_name"], has_string("Michael"), reason="Wrong name")
```

Comment:
In all assertions you use the same structure `response.json()["data"]`. What do you think if we will set a special variable to get the 'data' tag only one time?

e.g:
response = requests.get(url=get_list_users_url)
data = response.json()["data"][0]
assert_that(data["email"], is_(equal_to("michael.lawson@reqres.in")), reason="Wrong email")
assert_that(data["first_name"], has_string("Michael"), reason="Wrong name")